utils_cv/tracking/model.py

```diff
@@ -192,6 +193,8 @@ def fit(
                 for param_group in optimizer.param_groups:
                     param_group["lr"] = lr
 
+        self.save(self.model_path)
```

Comment:
Can you describe here a bit more why we save (because the original functionality assumes the model being written to disk etc)